src/mmw/apps/home/urls.py

```diff
@@ -15,4 +15,5 @@
     url(r'^analyze$', home_page, name='analyze'),
     url(r'^compare$', compare, name='compare'),
     url(r'^error', home_page, name='error'),
+    url(r'^register', home_page, name='register'),
```

Comment:
This changeset demonstrates why trailing commas are useful. Had the previous line not had a trailing comma, it would also show up as changed.